### ISSUE TITLE
Last line of multiline labels invisible

### ISSUE THREAD

**Issue by turdparty** [haxeui/haxeui-core#461]
I'm using the lastest git version of haxeui through openfl test html5.

The issue is that the last line of a multiline label won’t get displayed.
It does increase the height, but not quite enough. Probably does not account for margins, spacing, padding, whichever...

In the upper label it should show “some text\n123”. With some extra space it does.

![](https://community.haxeui.org/uploads/default/original/1X/8994f6a8be6ffe1939926ef4997606ed002506e5.png)

**Comment by ianharrigan** [comment]
Are you on a recent version of openfl? I _believe_ the issue (if its the same im thinking of) is resolved there, can you also try the flag `openfl_textfield_workarounds` (ie, `-D openfl_textfield_workarounds`) and see if that makes things any better?

**Comment by turdparty** [comment]
Yes I'm on openfl 9.1.0.
With the flag added I get tons of syncvalidation issues, but I was able to get a test case in the end, and the error still occurred. No difference.

**Comment by ianharrigan** [comment]
Hmmm, ok, any chance of a test app? I think openfl git might fix it, though i appreciate that is likely something you dont want to do. I can test this end against openfl git if you have a semi small test app?

**Comment by turdparty** [comment]
I made a simple test app. Unzip and run it using openfl test html5. Then in the window, keep clicking the "bla" text, it will keep adding text. As soon as the text wraps, the last line of text will not be shown, until another line is added etc.

[software.zip](https://github.com/haxeui/haxeui-core/files/8926990/software.zip)

**Comment by ianharrigan** [comment]
So, i think this is fixed in later versions of openfl, im on git, quite an old commit, but certainly not as old as the last official release, which i believe is 1year+ old. As far as a know a new version will be coming out _very_ soon, so maybe lets leave this open and revisit it when that happens?

Also, just to sanity check, this is the behaviour i saw with your test app, you are saying you dont get this behaviour, correct?

![openfl-wrapping](https://user-images.githubusercontent.com/3024242/174430147-32dab93a-f3c3-4990-b63f-53591a8f83e8.gif)

**Comment by turdparty** [comment]
Correct, the second line only shows up when the third line is started, and then the third line being invisible.

Alright yeah, let's leave it open.
